test/spmd/test_dynamo_spmd.py

```diff
@@ -33,7 +34,7 @@ class DynamoSpmdInferenceTest(test_xla_sharding_base.XlaShardingTest):
 
   @classmethod
   def setUpClass(cls):
-    os.environ["XLA_USE_SPMD"] = "1"
+    xr.use_spmd()
```

Comment:
One thought, we need api to query whether spmd is being used on python level. I am thinking something along the line of `xr.spmd_enabled`. given that would it be better to name this api to `xr.enable_spmd`? @alanwaketan @yeounoh

Reply:
I see, I can expose an API to check if it's SPMD. As SPMD is an execution "mode", I personally prefer simple API names like "is_spmd" & "use_spmd", but open to the alternatives, "spmd_enabled", "enable_spmd". @JackCaoG do you think it's better with "enable"?

Reply:
All tests are passing, I will wait for @JackCaoG @alanwaketan 's feedback on the naming.